### ISSUE TITLE
Meta-Issue: Active PR/issues

### ISSUE THREAD

**Issue by fundamental** [zynaddsubfx/zyn-fusion-issues#272]
This is an experiment in more publicly showing what's on my todo queue. We'll see if this works out or if updating this takes a bit too much time. Ideally these are roughly ordered based upon priority, but very small PRs which are quick to check will tend to skip priorities:

- https://github.com/mruby-zest/mruby-zest-build/issues/31
- https://github.com/mruby-zest/mruby-zest-build/pull/32 [DONE]
- https://github.com/zynaddsubfx/zynaddsubfx/pull/85
- https://github.com/mruby-zest/mruby-zest-build/issues/33
- https://github.com/zynaddsubfx/zyn-fusion-build/pull/71
- https://github.com/zynaddsubfx/zyn-fusion-build/issues/70 [could not replicate]
- https://github.com/mruby-zest/mruby-zest-build/issues/35 [DONE for now]
- https://github.com/mruby-zest/mruby-zest-build/pull/36 [DONE]
- https://github.com/mruby-zest/mruby-zest-build/pull/37 [deferred]
- https://github.com/fundamental/rtosc/pull/52
- https://github.com/mruby-zest/mruby-zest/pull/46

I'm guessing I'll have about an hour or so later today (Jan 30th), next timeslot has not been scheduled.

**Comment by fundamental** [comment]
Looks like today should be a good day for some house keeping, though given I'm somewhat braindead today, no complex reviews/debugging.

**Comment by fundamental** [comment]
2 PRs merged, 1 issue updated.

**Comment by fundamental** [comment]
I've got some other project work for the later part of the evening, but I should be able to get through a bit of the backlog now. Perhaps it will be possible to update the current todo list.

**Comment by fundamental** [comment]
1 support email, 5+ issue comments, 1 undocumented bug fix, 1 PR tested+merged.

**Comment by fundamental** [comment]
Looks like I'll target a lot of the small issues this morning, then late this evening try to rebuild the current work log. Most of the items there are expected to be large scale tasks involving a lot of review.

**Comment by fundamental** [comment]
2 issue comment, 6 PR merged, 1 issue closed. Might be a short term record.

**Comment by fundamental** [comment]
Spending some time (~1hr) tonight to get the queue back in order.

**Comment by fundamental** [comment]
- https://github.com/zynaddsubfx/zynaddsubfx/pull/85 Zyn Ladder PR. This one is on the meatier side. I'm planning on addressing while testing out a physical ladder circuit of mine.
- https://github.com/zynaddsubfx/zyn-fusion-build/pull/71 Conversion of build to Makefiles. This needs extensive testing to be sure as after merging I would deprecate the old code.
- https://github.com/mruby-zest/mruby-zest-build/pull/41 High priority merge of osc-bridge and mruby-zest into mruby-zest-build. May be a blocker for other pull requests, so this should be addressed soon.
- https://github.com/zynaddsubfx/zynaddsubfx/pull/113 Fast to review bug fixes [DONE]
- https://github.com/zynaddsubfx/zyn-fusion-issues/issues/274 issue which looks like it will be unblocked after mruby-zest-build:41
- https://github.com/zynaddsubfx/zyn-fusion-issues/issues/275 issue needing replication tests
- https://github.com/zynaddsubfx/zyn-fusion-issues/issues/276 issue needing elaboration to direct future PR(s) [ELABORATED]
- https://github.com/zynaddsubfx/zynaddsubfx/pull/108 check on rtosc changes? [PENDING submodule update]

**Comment by friedolino78** [comment]
> * [zynaddsubfx/zynaddsubfx#108](https://github.com/zynaddsubfx/zynaddsubfx/pull/108) check on rtosc changes?
look at https://github.com/fundamental/rtosc/pull/53
